.github/workflows/testground.yml

```diff
@@ -52,8 +64,38 @@ jobs:
          -b=docker:go -r=local:docker \
          -tp=numOfHubs=2 -tp=numOfPayers=2 -tp=numOfPayees=2  -i=6 \
          -tp=paymentTestDuration=30  -tp=concurrentPaymentJobs=1 \
-          --metadata-repo "${{github.repository}}" \
-          --metadata-branch "${{github.ref}}" \
-          --metadata-commit "${{github.sha}}" 
+         --metadata-repo "${{github.repository}}" \
+         --metadata-branch "${{github.ref}}" \
+         --metadata-commit "${{github.sha}}"  | tee run.out
 
-      
\ No newline at end of file
+      # Parse the run id from the testground run output
+      - name: Set Run Id
+        id: set_run_id
+        run: | 
+         echo "::set-output name=RUN_ID::$(awk '/run is queued with ID/ {print $10}' <run.out)"
+      # Set the end time for the dashboard time range
+      - name: Set Done Time
+        id: set_done_time
+        run: | 
+         echo "::set-output name=DONE_TIME::$((`date '+%s'`*1000))"
+      
+      # Look for an existing comment from the bot and update it 
+      - name: Find dashboard links comment
+        uses: peter-evans/find-comment@v2
+        id: find-comment
+        with:
+         issue-number: ${{ github.event.pull_request.number }}
+         comment-author: 'github-actions[bot]'
+         body-includes: Testground Run
+      - name: Update dashboard links comment
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          comment-id: ${{ steps.find-comment.outputs.comment-id }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            # 🧪 Testground Run for ${{github.event.pull_request.head.sha}} 
+            - [Message Stats Dashboard](http://34.168.92.245:3000/d/miulKz7Vk/message-stats?orgId=1&var-runId=${{steps.set_run_id.outputs.RUN_ID}}&from=${{steps.set_start_time.outputs.START_TIME_OFFSET}}&to=${{steps.set_done_time.outputs.DONE_TIME}})
```

Comment:
Do you think it is worth defining the IP address and port(s) as constants somewhere (perhaps even github secrets)? Might help us to refactor in future.

Reply:
Updated in https://github.com/statechannels/go-nitro/pull/895/commits/b859f86b1b4c432b81178667c08adb77088ea287